src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1544,6 +1555,271 @@ emit_vector64_vector128_t (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 
 #endif // defined(TARGET_AMD64) || defined(TARGET_ARM64)
 
+#ifdef TARGET_AMD64
+
+// System.Numerics.Vector2/Vector3/Vector4
+static guint16 vector2_methods[] = {
```

Comment:
Maybe name it so its more clear when used that its the same methods used for Vector2/3/4?